### ISSUE TITLE
[nl] text issues on site

### ISSUE THREAD

**Issue by baarsrj** [languagetool-org/languagetool#7822]
The normal word for spell check in Dutch is 'spellingcontrole'. The exra s is not wrong though, but not very common.
All languages are reported as spell checking (Why?), only Dutch is 'grammar check'.
The space in 'interpunctie controle' is very wrong. And the space in Word documenten just as mistaken.
The hyphens/dashes in open-source-ontwikkeling are also optional, so not the base form of the word, but really wrong either.

![afbeelding](https://user-images.githubusercontent.com/8479998/218271145-9e1f9105-dea9-4e2e-a294-a68ef985b679.png)

**Comment by danielnaber** [comment]
> The space in 'interpunctie controle' is very wrong. And the space in Word documenten just as mistaken.
> The hyphens/dashes in open-source-ontwikkeling are also optional, so not the base form of the word, but really wrong either.

Just to confirm, are these the correct spellings?
* spellingcontrole
* interpunctiecontrole
* Worddocumenten
* opensourceontwikkeling

**Comment by baarsrj** [comment]
Yes, they are.Op 11 feb. 2023 20:00 schreef Daniel Naber ***@***.***>:

The space in 'interpunctie controle' is very wrong. And the space in Word documenten just as mistaken.
The hyphens/dashes in open-source-ontwikkeling are also optional, so not the base form of the word, but really wrong either.

Just to confirm, are these the correct spellings?
spellingcontroleinterpunctiecontroleWorddocumentenopensourceontwikkeling

**Comment by tiff** [comment]
I have made some corrections. We use "spellingscontrole" to satisfy Google

![Bildschirm­foto 2023-02-20 um 17 50 13](https://user-images.githubusercontent.com/37363/220163380-e3ab2217-bc3b-46f4-b6dc-d8de21262d0b.png)

**Comment by baarsrj** [comment]
When checking Google, it looks like spellingscontrole is written more frequently than spellingcontrole.Both forms are officially correct, because the s joining the parts is optional. But... the official rule is that you write the s when you hear it pronounced.I see it written a lot, but I never hear it pronounced that way.So I still think Google is wrong (simply becaus it is using bad text, internet text), and by suggesting the writte form with s, is pushing the crowd the wrong way. Like Microsoft does with suggestung spaces in words.But... since both forms are correct, and there is no winning from big tech, I will simply just let it rest.Op 20 feb. 2023 17:50 schreef Christopher Blum ***@***.***>:

I have made some corrections. We use "spellingscontrole" to satisfy Google
